projects/angular/src/wizard/wizard-stepnav-item.ts

```diff
@@ -158,4 +170,30 @@ export class ClrWizardStepnavItem {
       throw new Error('Wizard stepnav item is not associated with a wizard page.');
     }
   }
+
+  private ensureCurrentStepIsScrolledIntoView() {
+    const stepnavItemElement = this.elementRef.nativeElement;
+    const stepnavWrapperElement = stepnavItemElement.closest<HTMLDivElement>('.clr-wizard-stepnav-wrapper');
+
+    return this.navService.currentPageChanged.pipe(
+      tap(currentPage => {
+        if (
+          currentPage === this.page &&
+          !elementIsScrolledIntoView({ container: stepnavWrapperElement, element: stepnavItemElement })
+        ) {
+          stepnavItemElement.scrollIntoView({ behavior: 'smooth', block: 'center' });
+        }
+      })
+    );
+  }
```

Comment:
We should skip going into tap if the page isn't current so we don't have to execute "elementIsScrolledIntoView" for all the pages on each change
```suggestion
    return this.navService.currentPageChanged.pipe(
      filter(currentPage => currentPage === this.page),
      tap(currentPage => {
        if (!elementIsScrolledIntoView({ container: stepnavWrapperElement, element: stepnavItemElement })
        ) {
          stepnavItemElement.scrollIntoView({ behavior: 'smooth', block: 'center' });
        }
      })
    );
  }
```